src/components/RadioGroup/RadioButton.js

```diff
@@ -80,6 +91,25 @@ class RadioButton extends React.Component {
     LabelText: RadioGroupButtonLabelText,
   };
 
+  static Small = withProps({
+    Label: RadioGroupButtonLabel.Small,
+  })(RadioButton);
+  static Large = withProps({
+    Label: RadioGroupButtonLabel.Large,
+  })(RadioButton);
+
+  static Group = RadioGroupContainer;
+
+  renderChildren = () => {
+    const { children, checked } = this.props;
+
+    if (typeof children === 'function') {
```

Comment:
Does this actually work? I thought you needed to use the `React.children` API to handle children - I would expect this to be a weird wrapper around a function that you would need to unwrap with `React.children.only()`.

Reply:
`children` is just whatever you pass it, right? I guess if you passed a component class into `children`, this would break. But as a render prop I think it works.

Reply:
It looks okay when I look at it in `console.log`. Maybe I'm remembering past React issues...